data/human/news.txt

```diff
@@ -2150,3 +2150,324 @@ news "graffiti on spaceport wall"
+		word
+			"riots. "
+			"protests. %"
+		word
+			"The rioters are "
+		word
+			"singing, ${CCOR Song Description}. "
+			"chanting, a list of their demands including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"but they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% The rioters" "The protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet Alvorada
+name
+	"Observation"
+message
+	word
+		"You can't escape the feeling of being watched. It"
+	word
+		"'s probably the security cameras; most of them are still running."
+		" feels like there are eyes around every corner."
+		"'s the pirates; a small number of them seem to be observing you."
+	word
+		" After some time of this "
+	word
+		"a silent alarm tells you that your should check on your ship. "
+		"it may be wise make sure no-one's tampering with your ship. "
+	word
+		"You find "
+	word
+		"a lone pirate prodding the ship with rudimentary tools. "
+		"a few pirates watching but not apparently touching it. "
+		"no-one there. The area around your ship is almost suspiciously devoid of people. "
+	word
+		"Whatever is going on, your ship appears unscathed."
+
+news "Microgravity"
+	location
+		planet Mordente-Bridi
+name
+	"Repairs"
+message
+	word
+		"Watching the micro-gravity section of the spaceport from the bar, you "
+	word
+		"observe "
+		"notice "
+		"spot "
+		"spy "
+	word
+		"a Bastion "
+		"an Aerie "
+		"a Falcon "
+		"a Leviathan "
+	word
+		"returning from a raid, "
+	word
+		"potmarked with laser burns but with no apparent serious damage. "
+		"battered with thousands of small perforations. Somebody's turned a mainstay of the pirate arsenal, Gatling guns, back on them. "
+		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missile. "
+	word
+		"As it comes to a stop in the maintenance bay engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+		"They'll be working for the rest of the shift on this one."
+
+news "Symbol War"
+	location
+		planet Gagarin
+name
+	"Guerrilla Symbology"
+message
+	word
+		"It's an average day in Gagarin's busy spaceport, until "
+	word
+		"your attention is grabbed by an oddity. A "
+		"an unusual commotion catches your attention. A "
+	word
+		"squad "
+		"band "
+		"team "
+	word
+		"of CCOR "
+	word
+		"guerillas "
+		"insurgents "
+	word
+		"have entered the port."
+	word
+		"Almost none "
+		"Few "
+		"Some "
+		"All "
+	word
+		"of them are armed, but it's their other equipment that stands out; paint, plaster, and ladders. They "
+	word
+		"aren't disrupting traffic "
+		"are leaving port traffic be "
+	word
+		"as they "
```

Comment:
```suggestion
```
This bit seems irrelevant imo. Do you think it's necessary?

Reply:
Yeah, looking at it now that does seem unnecessary